tests/virt_autotest/login_console.pm

```diff
@@ -64,8 +64,11 @@ sub login_to_console {
             #assert_screen([qw(grub2 grub1)], 120);
 
             my $host_installed_version = get_var('VERSION_TO_INSTALL', get_var('VERSION', ''));
-            ($host_installed_version) = $host_installed_version =~ /^(\d+)/;
-            if ($host_installed_version eq '11') {
+            ($host_installed_version) = $host_installed_version =~ /^(\d+)/im;
+            my $host_upgrade_version = get_required_var('UPGRADE_PRODUCT');         #format sles-15-sp0
+            my $host_upgrade_major   = $host_upgrade_version =~ /sles-(\d+)-sp/i;
+            my $host_upgrade_minor   = $host_upgrade_version =~ /sp(\d+)$/im;
```

Comment:
Maybe the name can follow up general rule, like host_upgrade_release/host_upgrade_spack? Of course it is only a suggestion :-) .

Reply:
Is this you only concern ? I can make the change.

Reply:
yes, others LGTM

Reply:
Don't know why 'm' is needed to match the pattern but it is a minor issue. LGTM.